ale_linters/cs/omnisharp.vim

```diff
@@ -0,0 +1,31 @@
+" Author: Luiz Ribeiro <luizribeiro@gmail.com>
+" Description: C# support via omnisharp
+
+call ale#Set('cs_omnisharp_mono_executable', 'mono')
+call ale#Set('cs_omnisharp_executable', 'OmniSharp.exe')
```

Comment:
Try using just `OmniSharp`. Because of `PATHEXT`, you should be able to run it without `.exe` in the filename on Windows, if it's in `PATH`.